### DISCUSSION TITLE
[Prism.Maui] Overlapping region do not work properly in MacOS side

### DISCUSSION THREAD

**Discussion by hrahmansha** [PrismLibrary/Prism#2978] [.NET MAUI]
I am trying to initialize 3 regions in a certain area of my application. Region1 will have the entire area. Region2 and Region3 will split the area vertically.

![PrismMauiRegionTest](https://github.com/PrismLibrary/Prism/assets/19286332/3a26e7d8-36ac-4574-aee1-5ffe291485a0)

I am initializing the region in the MainPage.xaml something like this-

```
<ContentView Grid.Row="2" prism:RegionManager.RegionName="Region1"/>
<Grid Grid.Row="2">
      <Grid.ColumnDefinitions>
        <ColumnDefinition Width="*" />
        <ColumnDefinition Width="*" />
      </Grid.ColumnDefinitions>
      
      <ContentView Grid.Column="0" prism:RegionManager.RegionName="Region2"/>
      <ContentView Grid.Column="1" prism:RegionManager.RegionName="Region3"/>
</Grid>
``` 
   
Everytime I want to set view inside region, I clear all 3 regions and inject view in the preferred region.

```
_regionManager.Regions["Region1"].RemoveAll();
_regionManager.Regions["Region2"].RemoveAll();
_regionManager.Regions["Region3"].RemoveAll();
``` 
```
_regionManager.RequestNavigate(myPreferredRegion, myView);
``` 

This is working fine on Windows side. But in MacOS side, mouse click doesn't work on the view that is injected in the lower layer region.

I have attached a test project. Button in ViewA and ViewB is not working on MacOS side. Please check and let me know if there is anything I am missing.
[TestPrismMauiRegion.zip](https://github.com/PrismLibrary/Prism/files/12858388/TestPrismMauiRegion.zip)

Environment Info-
.Net7
Prism.DryIoc.Maui-8.1.119-beta
Windows 10, 11
MacOS 13.3.1

**Comment by dansiegel** [comment]
Your issue is a MAUI one not a Prism one. Prism does not control anything to do with the native platforms or views. If you did not have Prism Regions but instead just used the same sort of UI that you swapped out in your code behind you would see the same behavior. I would suggest that you create a sample like that and submit the issue to the MAUI team.
